spec/bolt/node/ssh_spec.rb

```diff
@@ -106,7 +107,122 @@ def result_value(stdout = nil, stderr = nil, exit_code = 0)
     end
   end
 
-  context "when running_commands" do
+  context "when executing with private key" do
```

Comment:
Is it important that we run all of these tests with both a private key and a password?

Reply:
probably not that important.